src/main/java/com/github/vkorobkov/jfixturescmd/CmdParser.java

```diff
@@ -0,0 +1,43 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParameterException;
+import com.github.vkorobkov.jfixtures.JFixtures;
+import com.github.vkorobkov.jfixtures.fluent.JFixturesResultImpl;
+import com.github.vkorobkov.jfixturescmd.utils.PropertiesReader;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import java.lang.reflect.Method;
+
+@Slf4j
+final class CmdParser {
+    private final CmdArgs cmdArgs = new CmdArgs();
+
+    void parse(final String[] args) {
+        final JCommander jCommander = JCommander.newBuilder()
+                .addObject(cmdArgs)
+                .programName(PropertiesReader.PROJECT_NAME)
+                .build();
+
+        try {
+            jCommander.parse(args);
+
+            if (args.length == 0 || cmdArgs.isHelp()) {
```

Comment:
What happens if we don't check `args.length == 0` but pass zero parameters to the app? Can we remove `args.length == 0` check?